.circleci/config.yml

```diff
@@ -70,6 +70,8 @@ jobs:
     <<: *e2eTest
   e2eTestWIPPRClusterState:
     environment:
+      IGNITION_DEBUG_ENABLED: "true"
+      IGNITION_DEBUG_LOGS_PREFIX: "wip-pr-clusterstate"
```

Comment:
Would everyone overwrite these or should we just have them enabled by default with a prefix like `azure-operator-wip-pr-clusterstate` so we don't need to worry about different operators and so on.

Also should inject a build number IMO.

Reply:
There is a clusterid included (might be changed to build number, but didn't find a clean way as there is no support for nested env variables) https://github.com/giantswarm/azure-operator/pull/641/files#diff-0a941e625c8bd762a7466d659e2664c2R43

Reply:
> Would everyone overwrite these or should we just have them enabled by default with a prefix like azure-operator-wip-pr-clusterstate so we don't need to worry about different operators and so on.

I thought about diffs here as for e.g. multiaz test you'll have a different prefix.